common/etl/buffers.go

```diff
@@ -0,0 +1,167 @@
+package etl
+
+import (
+	"bytes"
+	"sort"
+	"strconv"
+)
+
+const (
+	//SliceBuffer - just simple slice w
+	SortableSliceBuffer = iota
+	//SortableAppendBuffer - just simple slice w
+	SortableAppendBuffer
```

Comment:
it isn't a slice though, as I understand it is a map `k => [v1 v2 ... vn]

Reply:
Correct. It's map slice byte with fixed-length values.